### DISCUSSION TITLE
image augmentation

### DISCUSSION THREAD

**Discussion by DrMRizk** [AlexeyAB/darknet#8145] [Q&A]
Hello @AlexeyAB @WongKinYiu @cenit 
I understood from the published paper that the current version of YOLOv4 uses CutMix and Mosaic data augmentation, DropBlock regularization, Class label smoothing, etc.
Also, it is mentioned in the section "Improvements in this repository" that image augmentation has been added to YOLOv4. 

So,while training the model will augment the images automatically? Or we shall activate image augmentation before training?

we just need to modify the cfg file by choose
cutmix=1
mosaic=1
mixup=1 
blur=1

right?

note that  mixup and blur are not added to yolov4-custom.cfg;

**Comment by AlexeyAB** [comment]
@DrMRizk Hi, the model will augment the images automatically.

**Reply by DrMRizk** [reply]
Thank you for your rapid reply.
Which augmentation style would the model apply  automatically?

What is the role of :+1: #cutmix=1 and  mosaic=1 in the cfg file?
